libraries/gcp_backend.rb

```diff
@@ -17,6 +19,12 @@ def initialize(opts)
     @opts = opts
     # ensure we have a GCP connection, resources can choose which of the clients to instantiate
     @gcp = inspec.backend
+
+    # Magic Modules generated resources use an alternate transport method
+    # In the future this will be moved into the train-gcp plugin itself
+    if opts[:use_http_transport]
```

Comment:
How does this option get passed in? What happens if it doesn't get passed in?

Reply:
The option is passed in by MM generated resources that inherit this class. They call add that flag to their opts when they pass it to super().

If it isn't passed in we assume the resource does not use the REST APIs directly, and instead will use the SDKs, which is the current functionality.

Reply:
Ah, gotcha. That's a good plan.